src/NHSD.BuyingCatalogue.Identity.Api/Controllers/AccountController.cs

```diff
@@ -65,13 +70,19 @@ public async Task<IActionResult> Login(LoginViewModel viewModel)
 
             if (!signInResult.IsSuccess)
             {
-                if (signInResult.Errors.Contains(LoginUserErrors.UserNameOrPasswordIncorrect()))
+                var signInErrors = signInResult.Errors;
+
+                if (signInErrors.Contains(LoginUserErrors.UserNameOrPasswordIncorrect()))
                 {
                     ModelState.AddModelError(nameof(LoginViewModel.LoginError), SignInErrorMessage);
                 }
-                if (signInResult.Errors.Contains(LoginUserErrors.UserIsDisabled()))
+                if (signInErrors.Contains(LoginUserErrors.UserIsDisabled()))
```

Comment:
Can you add a blank line between the two `if`s? I think it's more readable with that spacing but up to you.

I'd probably think about inverting the second `if` but that's personal preference so entirely up to you.